swagger.yaml

```diff
@@ -805,11 +805,17 @@ components:
         - PENDING
         - UNDER_REVIEW
         - ACCEPTED
-        - DEPRICATED
+        - DEPRECATED
         - DECLINED
 
     SoftwareRegistry:
       type: object
+      required:
+        - name
+        - description
+        - url
+        - image
```

Comment:
Are you sure image should be required? Probably most things have an image but it seems to create a bit more work for the users- unless you are able to get the image straight from the Url (which would be very cool).

Reply:
image is required for the logo of the software